pyomo/contrib/solver/tests/solvers/test_ipopt.py

```diff
@@ -162,6 +152,97 @@ def test_version_cache(self):
         self.assertIsNone(opt._version_cache[0])
         self.assertIsNone(opt._version_cache[1])
 
+    def test_parse_output(self):
```

Comment:
Should there also be a test parsing an example of the "new-style" timing output?

Reply:
Thanks for the suggestion - I actually caught a bug because of this. It turns out that the old-style and new-style tables aren't the same either. Wheeeeee.